src/TodoList.jsx

```diff
@@ -0,0 +1,30 @@
+const todoList = [
+    {
+        id: 1,
+        title: 'Complete CTD Hawk assignment'
+    },
+    {
+        id: 2,
+        title: 'Call vet'
+    },
+    {
+        id: 3,
+        title: 'Cook family dinner'
+    }
+];
+
+const TodoList = () => {
+
+    return (
+        <>
```

Comment:
We use fragments when we need a single parent element for a component but don't want anything special. It's just an invisible box. In this case, we already have a single parent element in the `<ul>` element, so we can remove the fragment.